test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -533,7 +533,8 @@ export default class TabularView extends Component {
             let higherBetter;
             
             //get BenchmarkRouter & Metric files from server
-            metricProps = getMetricProps(parserProps, benchmark, metric);
+            let parserProps = getParserProps(benchmark);
```

Comment:
- add `await` when calling `getParserProps()`
- please update the comment above
- can we just use one line for checking `higherBetter` value? Something link following:
```
const higherBetter = parserProps.higherbetter ? parserProps.higherbetter : true;
```
In this way, we can remove line 533, line 538-line 546

Reply:
As to `higherbetter`values in `perfComapre`, since we need to decide whether or not to get rid of the `undefined` choice, I will leave it to next issue:https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/215